calico-enterprise_versioned_docs/version-3.17/releases.json

```diff
@@ -1,4 +1,265 @@
 [
+  {
+    "title": "v3.17.1",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.30.3",
+      "registry": "quay.io"
+    },
+    "calico": {
+      "minor_version": "v3.25",
+      "archive_path": "archive"
+    },
+    "components": {
+      "cnx-manager": {
+        "image": "tigera/cnx-manager",
+        "version": "v3.17.1"
+      },
+      "voltron": {
+        "image": "tigera/voltron",
+        "version": "v3.17.1"
+      },
+      "guardian": {
+        "image": "tigera/guardian",
+        "version": "v3.17.1"
+      },
+      "cnx-apiserver": {
+        "image": "tigera/cnx-apiserver",
+        "version": "v3.17.1"
+      },
+      "cnx-queryserver": {
+        "image": "tigera/cnx-queryserver",
+        "version": "v3.17.1"
+      },
+      "cnx-kube-controllers": {
+        "image": "tigera/kube-controllers",
+        "version": "v3.17.1"
+      },
+      "calicoq": {
+        "image": "tigera/calicoq",
+        "version": "v3.17.1"
+      },
+      "typha": {
+        "image": "tigera/typha",
+        "version": "v3.17.1"
+      },
+      "calicoctl": {
+        "image": "tigera/calicoctl",
+        "version": "v3.17.1"
+      },
+      "cnx-node": {
+        "image": "tigera/cnx-node",
+        "version": "v3.17.1"
+      },
+      "dikastes": {
+        "image": "tigera/dikastes",
+        "version": "v3.17.1"
+      },
+      "dex": {
+        "image": "tigera/dex",
+        "version": "v3.17.1"
+      },
+      "fluentd": {
+        "image": "tigera/fluentd",
+        "version": "v3.17.1"
+      },
+      "fluentd-windows": {
+        "image": "tigera/fluentd-windows",
+        "version": "v3.17.1"
+      },
+      "es-proxy": {
+        "image": "tigera/es-proxy",
+        "version": "v3.17.1"
+      },
+      "eck-kibana": {
+        "version": "7.17.9"
+      },
+      "kibana": {
+        "image": "tigera/kibana",
+        "version": "v3.17.1"
+      },
+      "eck-elasticsearch": {
+        "version": "7.17.9"
```

Comment:
This is updated to 7.17.11.

Reply:
@hjiawei This is just a test PR. But I've taken your inputs and updated the real PR: https://github.com/tigera/docs/pull/823